docs/integrations/prefect-ray/index.mdx

```diff
@@ -53,7 +53,7 @@ def shout(number):
 @flow(task_runner=RayTaskRunner)
 def count_to(highest_number):
     for number in range(highest_number):
-        shout.submit(number)
+        shout.submit(number).wait()
```

Comment:
i think we should use `.map` here, since `.wait()` like this would cause us to block `for` each item in the loop, which defeats the purpose of using `.submit` / `.map` for concurrency

```python
def count_to(highest_number):
    shout.map(range(highest_number)).wait()
```